debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -249,12 +249,12 @@ public void changeRecord(P partition,
                     LOGGER.warn(
                             "Error while processing event at offset {}",
                             changeRecordEmitter.getOffset().getOffset());
-                    break;
+                    throw new ConnectException("Error while processing event at offset " + changeRecordEmitter.getOffset().getOffset(), e);
```

Comment:
We don't want to throw new exception. We just want to modify this logging message and the one below to contain the full exception.